src/rules-generator.ts

```diff
@@ -33,8 +33,25 @@ var errors = {
   application: "Bolt application error: ",
 };
 
+/* TS does not allow for special properties to have distinct
```

Comment:
Yeah, it might make sense to pull the validator out as a class. Although feel free to do this another time